### DISCUSSION TITLE
Clan improvements/upgrades

### DISCUSSION THREAD

**Discussion by RoinujNosde** [RoinujNosde/SimpleClans#208] [Ideas]
Paying to unlock features

- More than one home
- More members
- Virtual chests
- Max balance in clan bank

**Comment by SrBedrock** [comment]
- Virtual Chests

**Comment by TomLewis** [comment]
An enderchest for Clans would be amazing.

**Comment by SrBedrock** [comment]
- Maximum amount that can be deposited in the bank of the clans, being able to give updrade to increase the maximum value.

**Comment by TomLewis** [comment]
I too run into an issue with infinite money in crew-banks this week, let us limit the maximum balance a crew bank can have!

**Comment by TomLewis** [comment]
1. Hook into GriefPrevention to `/trust TAG` Instead of `/trust clan.TAG` 
2. Some use for /clan war to do with Regions and fighting.

**Reply by RoinujNosde** [reply]
1. /clan land does something similar
2. #129
